k8s-artifacts/api-portal-with-analytics/api-portal/wso2apim-configmap.yaml

```diff
@@ -275,4 +275,16 @@ data:
     proxyPort = 32001
 
     [transport.http.properties]
-    proxyPort = 32002
\ No newline at end of file
+    proxyPort = 32002
+
+    [[event_listener]]
+    id = "token_revocation"
+    type = "org.wso2.carbon.identity.core.handler.AbstractIdentityHandler"
+    name = "org.wso2.is.notification.ApimOauthEventInterceptor"
+    order = 1
+
+    [event_listener.properties]
+    notification_endpoint = "https://wso2apim:32001/internal/data/v1/notify"
```

Comment:
We can use localhost for this right? No need to call this via the external lb.

Reply:
fixed with e6a5635